src/backend/templates/partials/footer.hbs

```diff
@@ -28,11 +28,16 @@
          {{#if copyright}}
          {{#if copyright.licence}}
          <p>
-         <a href="{{{copyright.licence_url}}}"> <img src="{{{copyright.logo}}}"> </a>
-         &copy; {{copyright.year}}
+        <a href="{{{copyright.licence_url}}}"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by/4.0/80x15.png" /></a>
```

Comment:
We can not hard code the Image URL as the licence logo may not necessarily be creative commons, it will vary according to the event. I would strongly recommend you to check this by creating a sample event here  https://open-event-dev.herokuapp.com for better understanding.